scam-detector-py/src/agent.py

```diff
@@ -936,6 +938,110 @@ def emit_new_fp_finding(w3) -> list:
 
     return findings
 
+
+# This function is called once per hour to update the REACTIVE_LIKELY_FPS dictionary. 
+# It fetches scammer addresses from Scam Detector alerts generated 89 days ago (at that specific hour) and then retrieves 
+# scammer labels for those addresses. The retrieved data is used to update the dictionary.
+def update_reactive_likely_fps(current_date) -> list:
+    logging.info(f"{BOT_VERSION}: update reactive likely fps called")
+
+    global REACTIVE_LIKELY_FPS
+
+    source_id = '0x47c45816807d2eac30ba88745bf2778b61bc106bc76411b520a5289495c76db8' if Utils.is_beta() else '0x1d646c4045189991fdfd24a66b192a294158b839a6ec121d740474bdacb3ab23'
+    response = None
+    page_size = 2000
+    starting_cursor = None
+    should_retry_from_error = False
+   
+    # Calculate the time 89 days ago (Querying alerts from 90 days ago results in an error)
```

Comment:
so if we add a scammer label and its clear it is an FP after 2 days, we wont correct the FP until 89 days? Why such a long timeframe?

Reply:
@christian-forta, querying and processing _all_ past scammer labels let's say each day would take forever as the number of labels was quite large. Because of that, I thought of splitting them into "hours periods" of X days ago. 

I chose "89" as the safest option, but now thinking about it again, it's definitely too much.  With the current implementation, perhaps 2 would be too short (as `Utils.is_fp` is also called during the original alert creation), so maybe I should change it to 7? 

Of course, I could also change the implementation if there's a more suitable approach.

Reply:
do you think it would make sense to have multiple period, like at 7 days, 30 days, 89 days? And review one hour each?